www/src/Pages/PinMapping.tsx

```diff
@@ -63,6 +72,47 @@ const PinsForm = ({ savePins, pins, setPinAction }: PinsFormTypes) => {
 		}
 	};
 
+	const pinList = useMemo<PinList>(() => {
+		const pinArray = Object.entries(pins);
+		return zip(
+			pinArray.slice(0, pinArray.length / 2),
+			pinArray.slice(pinArray.length / 2, pinArray.length),
+		);
+	}, [pins]);
+
+	const createCell = useCallback(
+		([pin, pinAction]: PinCell) => (
+			<div className="d-flex col py-2">
+				<div className="d-flex align-items-center" style={{ width: '4rem' }}>
+					<label htmlFor={pin}>{pin.toUpperCase()}</label>
+				</div>
+				<Select
+					inputId={pin}
+					className="text-primary flex-grow-1"
+					isClearable
+					isSearchable
+					options={options}
+					value={getOption(pinAction)}
+					isDisabled={isNonSelectable(pinAction)}
+					getOptionLabel={(option) => {
+						const labelKey = option.label.split('BUTTON_PRESS_').pop();
+						// Need to fallback as some button actions are not part of button names
+						return (
+							buttonNames[labelKey] || t(`PinMapping:actions.${option.label}`)
+						);
+					}}
+					onChange={(change) =>
+						setPinAction(
+							pin,
+							change?.value === undefined ? -10 : change.value, // On clear set to -10
+						)
+					}
+				/>
+			</div>
+		),
+		[],
```

Comment:
`options` technically is a dependency but there are no practical differences since it is a const

Reply:
Thanks for checking the pr out!

I believe that options is not a dependency as it's not a reactive value, it's defined outside of the component

This even matching and exact example from the new react docs on how to use dependency array with non reactive value: 
From [react.dev](https://react.dev/)
```javascript
const serverUrl = 'https://localhost:1234';
const roomId = 'general';

function ChatRoom() {
  useEffect(() => {
    const connection = createConnection(serverUrl, roomId);
    connection.connect();
    return () => {
      connection.disconnect();
    };
  }, []); // ✅ All dependencies declared
  // ...
}
```

Source: https://react.dev/learn/lifecycle-of-reactive-effects#what-an-effect-with-empty-dependencies-means